jina/docker/checker.py

```diff
@@ -68,7 +68,7 @@ def remove_control_characters(s):
 
 
 def safe_url_name(s):
-    return s.lower().replace('-', '--').replace('_', '__').replace(' ', '_')
+    return s.lower().replace('_', '__').replace(' ', '_')
```

Comment:
is there a unittest on what is expected here? why did we remove that part?

Reply:
It was never used before and it was unnecessary (as discussed with @hanxiao ). The unittest in https://github.com/jina-ai/jina/pull/1341/files#diff-e63749458c2c70804a69ac37c0a560e869b070408233b988a219514a8df6cf31R55 covers it (the name format)